### ISSUE TITLE
Fatal error on `update` when attempting to expose a folder that does not exist (on Windows)

### ISSUE THREAD

**Issue by NightJar** [silverstripe/vendor-plugin#28]
# User story

Picture this if you will:

A developer sits, quietly attempting to extend a module on their corporate supplied windows machine. A simple skeleton has been laid out, including some front-end build dependencies which will later be generated by webpack.
Therefore the folder `client/dist` does not yet exist (although `client/src` does).

Attempting to require this module as a composer dependency fails with a fatal error when composer attempts to expose `client/dist`.

>But dear developer, you are the module maintainer. Just create the dist folder by building the assets!

Alas, the developer's module has requirements on admin - which requires node 6.
Until recently, when now it requires node 11+ (because of the React 16 update), which is what they have installed. An upgrade will mean that the module can only support being _built_ with newer versions of SilverStripe (that do not yet exist).
After a little ado getting the windows version of NVM operating in order to press on, they are disappointed that downgrading does not work, due to the admin constraints - but this is merely frustrating and by the bye to this issue.

The developer feels disheartened that absolutely nothing works, and perhaps they should just pay for an ad free wix site instead... But suddenly in a last ditch attempt the developer flips their table, breaks open a VM, fevered flailing at their laptop. This VM uses a Linux kernel to power the guest using PHP 7.1 to run composer...

Which installs the dependency without any fatal error whatsoever, and everything works just fine.

The developer then throws their laptop into the sea and applies for a new job as a tour guide.

# TL;DR

> Your user story is rubbish @NightJar - too long and filled with irrelevant plot points... are you the person secretly behind episodes 100-3499 of Naruto?

No, sadly inner voice I am not. The tale of our fictitious user above is loosely based on a true story, believe it or not.

**As a Windows developer, I expect `vendor-expose` to operate as it does in a GNU+Linux environment**
Currently a fatal error is generated when attempting to expose a folder that does not exist.

I feel this is a poor developer experience, for those on the upgrade path.

Info:
Windows
PHP 5.6

I can switch to PHP 7.1 also and try again.

**Comment by StephenMakrogianni** [comment]
FYI: This is what it looks like

![image](https://user-images.githubusercontent.com/42820868/55930768-fd796b00-5c76-11e9-83b8-f0c880bf051c.png)
